doc/backend/setting.md

```diff
@@ -85,6 +86,132 @@ ldap_tls = False
 ⚠ When logging in Tracim, if a valid LDAP user doesn't
 exist in Tracim, it will be created as a standard user.
 
+## SAML Authentication
```

Comment:
I suggest to move this to end of authentication related stuff.
API-Key will be a more common use case than SAML, so SAML can be moved at the end of section.